tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/resource/PetResource.java

```diff
@@ -191,33 +194,37 @@ public Response addPet(Pet pet) {
-    @APIResponse(responseCode = "200",
+    @APIResponse(responseCode = "200", description = "Pets found with matching status",
                  content = @Content(mediaType = "application/json",
                                     schema = @Schema(type = SchemaType.ARRAY, implementation = Pet.class)))
     @Operation(summary = "Finds Pets by status",
                description = "Find all the Pets with the given status; Multiple status values can be provided with comma seperated strings")
     @Extension(name = "x-mp-method1", value = "true")
     @Extensions({@Extension(name = "x-mp-method2", value = "true"), @Extension(value = "false", name = "x-mp-method3")})
     public Response findPetsByStatus(
-            @Parameter(name = "status", description = "Status values that need to be considered for filter",
-                       required = true, schema = @Schema(implementation = String.class), content = {
-                               @Content(examples = {
-                                       @ExampleObject(name = "Available", value = "available",
-                                                      summary = "Retrieves all the pets that are available"),
-                                       @ExampleObject(name = "Pending", value = "pending",
-                                                      summary = "Retrieves all the pets that are pending to be sold"),
-                                       @ExampleObject(name = "Sold", value = "sold",
-                                                      summary = "Retrieves all the pets that are sold")
-                               })
+            @Parameter(name = "status", in = ParameterIn.QUERY,
+                       description = "Status values that need to be considered for filter",
+                       required = true, content = {
+                               @Content(
+                                        schema = @Schema(implementation = String.class),
+                                        examples = {
+                                                @ExampleObject(name = "Available", value = "available",
+                                                               summary = "Retrieves all the pets that are available"),
+                                                @ExampleObject(name = "Pending", value = "pending",
+                                                               summary = "Retrieves all the pets that are pending to be sold"),
+                                                @ExampleObject(name = "Sold", value = "sold",
+                                                               summary = "Retrieves all the pets that are sold")
+                                        })
                        }, allowEmptyValue = true) @Extension(name = "x-mp-parm1", value = "true") String status) {
         return Response.ok(petData.findPetByStatus(status)).build();
     }
```

Comment:
This whole method looks fishy. There's no JAX-RS param annotation on the `status` parameter, so it should be the request body, which for a `GET` request is questionable at best.

I think an implementation could reasonably throw an exception if a `@Parameter(in=Query)` annotation is attached to a request body parameter, so I don't think this is valid.

I can't see any test that asserts anything about this method so I would advocate to remove it in a micro release instead of trying to fix it.

Reply:
> I can't see any test that asserts anything about this method so I would advocate to remove the whole method in a micro release instead of trying to fix it.

I agree here as well. I was trying to minimize changes, but if it's fine to do it in a micro release, taking this method out would be better.

Reply:
Removed the method.